internal/provider/resource_environment.go

```diff
@@ -223,6 +223,34 @@ func resourceEnvironment() *schema.Resource {
 					},
 				},
 			},
+			"repositories": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"database_type": {
```

Comment:
Is this always going to be "postgres"? Or what are some examples of this?

Reply:
No this can be other than Postgres, like Oracle , ASE , PostgrtesSQL

Reply:
Ah, I didn't realize this resource was broad like that. We will definitely need to list out all the options or make it easier for customers to add one.
I thought we only added the Postgres API though?

Reply:
Maybe I missed updating here,  this new block I have added  "repositories" It is read-only , when we use environment resource the state file will now contain "repositories" also because it is already available in the response we were ignoring it till now , but we need it from now on for the source API,all the attributes under "repositories"  will be read-only similar to any get response.

Reply:
Ahhh, I didn't realize these were all output properties. That's on me. Okay, a lot of my comments can be ignored then. Sorry for the confusion.

Reply:
Np